.github/workflows/R-CMD-check.yaml

```diff
@@ -97,8 +114,9 @@ jobs:
           name: ${{ runner.os }}-r${{ matrix.config.r }}-results
           path: check
 
-      # TODO: uncomment this when we fix the failure of covr::codecov()
-      # - name: Test coverage
-      #   if: matrix.config.os == 'macOS-latest' && matrix.config.r == '3.6'
-      #   run: covr::codecov()
-      #   shell: Rscript {0}
+      - name: Test coverage
```

Comment:
Might be worthwhile to remove in favour of a separate action (that's what we're using elsewhere)

Reply:
Thanks, sounds nice! I'll remove this step and add https://github.com/r-lib/actions/blob/master/.github/workflows/test-coverage.yaml.